data_objects/lib/data_objects/ext/do_common.c

```diff
@@ -210,18 +210,20 @@ VALUE data_objects_parse_time(const char *date) {
     return Qnil;
   }
 
+  // TODO: support the timezone being returned from 
   return rb_funcall(rb_cTime, rb_intern("local"), 7, INT2NUM(year), INT2NUM(month), INT2NUM(day), INT2NUM(hour), INT2NUM(min), INT2NUM(sec), INT2NUM(usec));
 }
 
 VALUE data_objects_parse_date_time(const char *date) {
-  static char const* const _fmt_datetime_tz_normal = "%4d-%2d-%2d%*c%2d:%2d:%2d%3d:%2d";
-  static char const* const _fmt_datetime_tz_subsec = "%4d-%2d-%2d%*c%2d:%2d:%2d.%*d%3d:%2d";
+  static char const* const _fmt_datetime_tz_normal = "%4d-%2d-%2d%*c%2d:%2d:%2d%7lf%3d:%2d";
+  // static char const* const _fmt_datetime_tz_subsec = "%4d-%2d-%2d%*c%2d:%2d:%2d.%*d%3d:%2d";
```

Comment:
I don't think it was actually used. Maybe an attempt to handle this case that never was finished?